controls/3.7/def.cf

```diff
@@ -236,8 +236,9 @@ bundle common def
       # Internal CFEngine log files rotation
       "cfengine_internal_rotate_logs" expression => "any";
 
-      # Enable agent email output (also see mailto, mailfrom and smtpserver)
+      # Enable or disable agent email output (also see mailto, mailfrom and smtpserver)
       "cfengine_internal_agent_email" expression => "any";
+      "cfengine_internal_disable_agent_email" expression => "!any";
```

Comment:
If people are not expected to edit ``def.cf``, can't we just kill ``cfengine_internal_agent_email`` altogether? Or do we need it to be around for people with existing ``def.cf``?